docs/usage.rst

```diff
@@ -2,4 +2,204 @@
 Usage
 =====
 
-…
+:ref:`Given an HTML form <form>`:
+
+.. _parsel-example:
+
+>>> from parsel import Selector
+>>> html = b"""<form><input type="hidden" name="foo" value="bar" /></form>"""
+>>> selector = Selector(body=html, base_url="https://example.com")
+>>> form = selector.css("form")
+
+You can use :func:`~form2request.form2request` to generate :ref:`form
+submission request data <request>`:
+
+>>> from form2request import form2request
+>>> form2request(form)
+Request(url='https://example.com?foo=bar', method='GET', headers=[], body=b'')
```

Comment:
As a first-time user, I'd probably want to make the request. What do you think about adding a note right here, to make the example complete? Something about form2request not making http calls itself, and a link to a section for requests/scrapy (or even an example).